mpas_analysis/shared/generalized_reader/generalized_reader.py

```diff
@@ -176,6 +181,13 @@ def open_multifile_dataset(fileNames, calendar, config,
     # select only the data in the specified range of dates
     ds = ds.sel(Time=slice(startDate, endDate))
 
+    # process chunking
+    if chunking is True:
+        # limit chunk size to prevent memory error
+        chunking = config.getint('input', 'maxChunkSize')
```

Comment:
@pwolfram, this is definitely not the expected behavior.  `chunking` is getting overridden by the value in `input` even when we try to set a specific value in the MOC.  I think this has to be a bug.